src/pybind/mgr/.gitignore

```diff
@@ -1 +1,2 @@
 proxy.conf.json
+__pycache__
```

Comment:
Is there any reason not to add this to the top level .gitignore?

Reply:
I only have __pycache__ issues in /src/pybind/mgr, so it does not make sense to me to add it in an upper level (and i will not add it there if i do not know if there will be any issue ever).

Reply:
```
/ceph $ grep __pycache__ $(find * -name '.gitignore')
qa/.gitignore:__pycache__
src/pybind/.gitignore:/__pycache__
src/pybind/mgr/dashboard/.gitignore:__pycache__
src/seastar/.gitignore:__pycache__/
```
We already have `__pycache__` in `src` and `qa`. Would it make sense to unify it into the root .gitignore?